ir_cdk_stacks/in_clw_01/clwUnauthAccessResponse.py

```diff
@@ -0,0 +1,127 @@
+import json
+import boto3
+import urllib3
+import os
+
+iam = boto3.client('iam')
+sns = boto3.client('sns')
+#clwDenyPolicy = 'arn:aws:iam::544820149332:policy/ClWDenyAccess1'
+#clwDenyPolicy2 = 'arn:aws:iam::544820149332:policy/ClWDenyAccess2'
+clwDenyPolicyName1 = 'ClWDenyAccess1'
```

Comment:
This should be dynamic, try injecting environment variables into the lambda. You can check others code.

Reply:
Yes the ARN is dynamic.

Reply:
I have created explicit deny policy whose name should not be changed. Only the Arn can change so that is dynamic